src/realmGeolOrogen.ttl

```diff
@@ -151,23 +157,51 @@ soreagor:Orogen rdf:type owl:Class ;
                              owl:onProperty sorelph:hasPlanetaryStructure ;
                              owl:someValuesFrom sorealt:ThrustSystem
                            ] ;
-           rdfs:label "orogen"@en .
-
+           dcterms:source <http://www.glossaryofgeology.org> ;
+           dcterms:contributor <http://linked.data.gov.au/def/gsq> ;                           
+           skos:definition "An orogen (orogenic system, orogenic zone, orogenic belt) is a region in Earth’s lithosphere where a mountain belt is created by tectonic processes involving deformation, regional metamorphism, and associated magmatism, usually caused by convergence (± accretion and/or collision) between two tectonic plates or major crustal blocks. (Alt obsolete term: geosyncline). (Modified from Neuendorf et al., 2011; Hoy, pers. comm.)"@en ;                      
```

Comment:
An orogen is a part of the  Earth’s crust deformed by tectonic processes involving large-scale faulting, regional metamorphism, and associated magmatism, producing significant vertical displacements of rock bodies.

ScopeNote: usually caused by convergence (± accretion and/or collision) between two tectonic plates or major crustal block